aerogear/mobile-developer-console
# fix: remove extra watch for MSS namespace

## Motivation

https://issues.jboss.org/browse/AEROGEAR-9587

## What

Removed the steps to watch the Mobile Security Service namespace for the creation of the custom resource.

## Why

This was unnecessary as the installer will handle this.

## Verification Steps
Add the steps required to check this change. Following an example.
 
1. Ensure MDC is up and running.
2. Install the Mobile Security Service operator. You can use [this script](https://gist.github.com/craicoverflow/87f73bb0635c6552ef57e1a61f80d9e9) to install it yourself to your cluster.
3. Create an app in MDC. 
4. Bind the Mobile Security Service to the app. The Mobile Security Service URL should appear and the service should be added to `mobile-services.json`.

Contact me if you would prefer access to my remote cluster so that you can skip steps 1 and 2.

## Checklist:


## Progress
